tutorial/cdn-caching.md

```diff
@@ -0,0 +1,158 @@
+---
+title: CDN Caching
+nav_order: 22
+parent: Malibu Tutorial
+---
+
+# {{page.title}}
+
+_This tutorial was contributed by [Harshith](ttps://www.linkedin.com/in/harshith-raj-092ba4176)_
+
+How Quintype handles CDN caching?
+
+- Whenever a collection/story/author is created or updated, the bold handles cache/purging through the CDN.
+- When a collection/story/author is created or updated, the frontend gets updated with the latest data with the help of cache-keys.
```

Comment:
This is still vague, mention "the frontend gets updated with the latest data with the help of cache-keys returned in the API for that collection/story/author"

Reply:
sure